L1Trigger/RPCTriggerPrimitives/plugins/RPCProcessor.h

```diff
@@ -0,0 +1,85 @@
+#ifndef L1Trigger_RPCTriggerPrimitives_RPCProcessor_h
+#define L1Trigger_RPCTriggerPrimitives_RPCProcessor_h
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DataFormats/MuonDetId/interface/RPCDetId.h"
+#include "DataFormats/RPCDigi/interface/RPCDigiCollection.h"
+#include "DataFormats/RPCRecHit/interface/RPCRecHit.h"
+
+#include "DataFormats/RPCRecHit/interface/RPCRecHit.h"
+#include "DataFormats/RPCRecHit/interface/RPCRecHitCollection.h"
+
+#include "Geometry/RPCGeometry/interface/RPCRoll.h"
+#include "Geometry/RPCGeometry/interface/RPCGeometry.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+
+#include "RecoLocalMuon/RPCRecHit/interface/RPCRecHitAlgoFactory.h"
+
+#include "CondFormats/DataRecord/interface/RPCMaskedStripsRcd.h"
+#include "CondFormats/DataRecord/interface/RPCDeadStripsRcd.h"
+#include "CondFormats/RPCObjects/interface/RPCMaskedStrips.h"
+#include "CondFormats/RPCObjects/interface/RPCDeadStrips.h"
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+#include <algorithm>
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <memory>
+#include <string>
+#include <sstream>
+#include <utility>
+#include <vector>
+#include <boost/cstdint.hpp>
```

Comment:
where is this used?

Reply:
#include <vector> is needed to fill map vectors and stuff like.

Reply:
vector lubreary is needed to fill map vectors and stuff like.